lib/resources/r4/charge_item_create.yaml

```diff
@@ -0,0 +1,386 @@
+---
+name: Parameters
+field_name_base_url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be Parameters
+    url: https://hl7.org/fhir/r4/json.html#resources
+
+  - name: parameter
+    required: 'Yes'
+    type: List of BackboneElement
+    description: Parameters defined by the operation.
+    action: create
+    url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: chargeItem
+    required: 'Yes'
+    type: id
+    description: Logical id of this artifact
+    action: create
+    example: |
+      {
+        'parameter': [
+           {
+              'name': 'chargeItem',
+              'resource': {
+              'resourceType': 'chargeItem',
+              'status': 'unknown',
+              'subject': {
+                'reference': 'Patient/178594204'
+               }
+            }
+         ]
+       }
+      }
+    url: https://hl7.org/fhir/chargeitem-definitions.html#ChargeItem
+
+  - name: chargeItem.resourceType
```

Comment:
@SantanuBanerjee  the name is in correct format or it should be only resourceType